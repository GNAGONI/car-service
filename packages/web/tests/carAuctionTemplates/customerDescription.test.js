import React from 'react'
import renderer from 'react-test-renderer'
import CustomerDescription from '../../components/carAuctionTemplates/customerDescription'

it('renders correctly', () => {
  const tree = renderer.create(<CustomerDescription />).toJSON()
  expect(tree).toMatchSnapshot()
})
