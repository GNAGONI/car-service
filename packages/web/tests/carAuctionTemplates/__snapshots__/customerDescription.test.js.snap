// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="info-box"
>
  <div
    className="row"
  >
    <div
      className="col-12 col-lg-9"
    >
      <h2>
        Customer Description:
      </h2>
      <p>
        
      </p>
    </div>
  </div>
</div>
`;
